models/values_script.m

```diff
@@ -0,0 +1,46 @@
+%% BSG data inport script
+
+%% INSTRUCTIONS
+% To run the model, there are 5 steps that must be followed:
+%
+% 1) Make sure to create a .csv file located in the same directory as this
+% script with the correct structure;
+%
+% 2)Run this script in order to define the variables in the Workspace;
+%
+% 3)Open the Simulink model and in MODELING -> Model Settings -> Data
+% Inport/Export -> Input make sure that 'dataStructure' is 
+% defined as the input;
+%
+% 4)Make sure to change the dimensions of all input blocks to match the
+% ones defined as '1'  in this script.
+%
+% 5)Run the model.
+
+%% Data Inport
+
+%reads data from .csv file located in the same directory as this script;
+data = readmatrix('data.csv');
```

Comment:
Suggestion: Create a Matlab function and add the filename as parameter to define a standalone application.

This would allow to call the application with any input and output file.

ref: https://www.mathworks.com/help/compiler/create-standalone-app-from-matlab-function.html